package.json

```diff
@@ -38,7 +38,7 @@
     ]
   },
   "jest": {
-    "collectCoverage": true,
+    "collectCoverage": false,
```

Comment:
Accidental commit - need to remove.
Coverage collection was preventing me from using debugging with jest in vs code, so I meant to temporarily disable.